bundles/org.openhab.binding.ekey/src/main/resources/OH-INF/i18n/ekey.properties

```diff
@@ -15,6 +15,8 @@ thing-type.config.ekey.protocol.label = Protocol
 thing-type.config.ekey.protocol.description = Can be RARE, MULTI or HOME depending on what the system supports
 thing-type.config.ekey.delimiter.label = Delimiter
 thing-type.config.ekey.delimiter.description = The delimiter is also defined on the ekey UDP converter - use the ekey configuration software to determine which delimiter is used or to change it. Another option is _ (underscore)
+thing-type.config.ekey.sourceIp.label = Source IP
+thing-type.config.ekey.sourceIp.description = IPv4 address of a received eKey udp packet. Can be different from the ipAddress when using NAT. (e.g. in Kubernetes)
```

Comment:
Wouldn't it be better to call this natIP instead of sourceIp ? In fact, it is not the source of the packets.